Client/mods/deathmatch/logic/luadefs/CLuaVectorGraphicDefs.cpp

```diff
@@ -257,12 +257,16 @@ bool CLuaVectorGraphicDefs::SVGSetUpdateCallback(CClientVectorGraphic* vectorGra
     return false;
 }
 
-std::variant<CLuaFunctionRef, bool> CLuaVectorGraphicDefs::SVGGetUpdateCallback(CClientVectorGraphic* vectorGraphic)
-{
-    auto func = vectorGraphic->GetUpdateCallback();
-
-    if (std::holds_alternative<CLuaFunctionRef>(func))
-        return func;
-
-    return false;
-}
+// This doesn't work, there is no conversion from CLuaFunctionRef to the Lua side, so the parser always uses bool.
```

Comment:
Addint an overload to `lua::Push` should be possible.
`CLuaFunctionRef` just holds a `lua_ref` to the function.
I can't remember the actual name of the function, but you can find it IIRC in `CLuaArguments::CallGlobal`.